release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -487,6 +487,15 @@ The following OpenShift CLI (`oc`) commands were removed with this release:
 
 Support for deploying custom schedulers manually has been removed with this release. Use the xref:../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[{secondary-scheduler-operator-full}] instead to deploy a custom secondary scheduler in {product-title}.
 
+[id="ocp-4-11-future-removals"]
+=== Future Kubernetes API removals
+
+The next minor release of {product-title} is expected to use Kubernetes 1.25. Currently, Kubernetes 1.25 is scheduled to remove several deprecated `v1beta1` and `v2beta1` APIs.
+
+See the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Deprecated API Migration Guide] in the upstream Kubernetes documentation for the list of planned Kubernetes API removals.
+
+See link:https://access.redhat.com/articles/6955985[ Navigating Kubernetes API deprecations and removals] for information on how to check your cluster for Kubernetes APIs that are planned for removal.
```

Comment:
It doesn't seem to make a difference in the preview, but there's an extra space at the start of the static text in the link.